payments-core/src/main/java/com/stripe/android/googlepaylauncher/GooglePayLauncher.kt

```diff
@@ -29,11 +29,11 @@ import java.util.Locale
 class GooglePayLauncher internal constructor(
     lifecycleScope: CoroutineScope,
     private val config: Config,
-    private val googlePayRepositoryFactory: (GooglePayEnvironment) -> GooglePayRepository,
     private val readyCallback: ReadyCallback,
     private val activityResultLauncher: ActivityResultLauncher<GooglePayLauncherContract.Args>,
+    private val googlePayRepositoryFactory: (GooglePayEnvironment) -> GooglePayRepository,
     analyticsRequestFactory: AnalyticsRequestFactory,
-    analyticsRequestExecutor: AnalyticsRequestExecutor = DefaultAnalyticsRequestExecutor()
+    analyticsRequestExecutor: AnalyticsRequestExecutor
```

Comment:
Just reordering to match the order of `GooglePayPaymentMethodLauncher`, no changes to public API.